### PR TITLE
Remove Microphysics module

### PR DESCRIPTION
**Pull request by trontrytel**
### Description

This removes the Microphysics module. We are using the CloudMicrophysics.jl package instead.

- [x] Code follows the [style guidelines](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) OR N/A.
- [x] Unit tests are included OR N/A.
- [x] Code is exercised in an integration test OR N/A.
- [x] Documentation has been added/updated OR N/A.

### CODE REVIEW

File: docs/src/Theory/Atmos/Microphysics_1M.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
+# Microphysics
```
Comment (charleskawczynski) [inline docs/src/Theory/Atmos/Microphysics_1M.md:1]: Should this live in CloudMocrophysics.jl?

Comment (trontrytel) [reply docs/src/Theory/Atmos/Microphysics_1M.md:1]: I think it's ok to keep here. It talks about what package we are using and how we are coupling it to our model state variables

Comment (trontrytel) [reply docs/src/Theory/Atmos/Microphysics_1M.md:1]: Meaning - I moved all the docs on how individual microphysics process rates are computed. But kept the docs on how those rates are coupled to total energy here.

**Review by charleskawczynski — APPROVED** [review]

**Comment by trontrytel** [comment]
Allright then. Let's see if the CI picks anything I missed!

**Comment by charleskawczynski** [comment]
Github actions are experiencing [issues](https://www.githubstatus.com/), so we can't really merge anything right now ☹️ (unless we do it manually)

**Comment by charleskawczynski** [comment]
Hopefully it'll be back up tomorrow

**Comment by trontrytel** [comment]
> Github actions are experiencing [issues](https://www.githubstatus.com/) right now, so we can't really merge anything right now ☹️ (unless we do it manually)

Lets wait till they get fixed then. It's not urgent

**Pull request merged**